github/github.go

```diff
@@ -33,7 +41,66 @@ func New(organization string, c *config.Config) *API {
 		organization,
 		tc,
 		client,
+		cache{validDuration: time.Duration(6 * time.Hour)},
+	}
+}
+
+// OrgStats represents basic stats about the configured organization's
+// repositories and activity
+type OrgStats struct {
+	Repositories int            `json:"repositories"`
+	Stargazers   int            `json:"stargazers"`
+	Topics       map[string]int `json:"topics"`
+	Languages    map[string]int `json:"languages"`
+
+	CommitTotal int               `json:"commit_total"`
+	CommitGraph map[time.Time]int `json:"commit_graph"`
+}
+
+// GetOrgStats collects basic stats about the configured organization's
+// repositories and activity
+func (api *API) GetOrgStats() (OrgStats, error) {
+	// Return cache while valid
+	if api.cache.statsExpiry.After(time.Now()) && api.cache.statsData != nil {
+		return *api.cache.statsData, nil
+	}
+
+	// Generate new stats
+	ctx := context.Background()
+	repos, _, err := api.Repositories.ListByOrg(ctx, api.organization, nil)
+	if err != nil {
+		return OrgStats{}, err
+	}
+
+	// Collect stats from repositories
+	stats := OrgStats{
+		Topics:      make(map[string]int),
+		Languages:   make(map[string]int),
+		CommitGraph: make(map[time.Time]int),
+	}
+	for _, r := range repos {
+		// Collect basic repository stats
+		stats.Repositories++
+		stats.Stargazers += r.GetStargazersCount()
+		stats.Languages[r.GetLanguage()]++
+		for _, t := range r.Topics {
+			stats.Topics[t]++
```

Comment:
This looks okay, but we'll have to be careful about how we represent the `Topics` and `Languages` information on the front end, as we're counting the number of times each topic and language is used and we don't want to confuse that with the number of _distinct_ topics and languages we use.

Reply:
Yeah, I'm still considering various visualization options - I might not even include some of this in the website quite yet, since our repositories are typically pretty lacking as far as `Topic` tags go. I was thinking a distinct language count, as well as some sort of word cloud thing with our most popular languages/topics.

Reply:
I really like the idea of showing off all the programming languages we use.